web/src/routes/SearchPage.tsx

```diff
@@ -102,6 +102,9 @@ const SearchPage = ({ city, cityCode, languageCode, pathname }: CityRouteProps):
       {query.length > 0 && (
         <>
           <List>
+            <p style={{ margin: '10px 5px', fontSize: 'small' }}>
```

Comment:
:wrench: Generally, we use styled-components wherever possible to style our components. Please adjust your PR accordingly. For examples, just search for `= styled`, e.g. in L20 above.
You can find the docu here: https://styled-components.com/docs/basics#getting-started